extensions/ipynb/src/notebookImagePaste.ts

```diff
@@ -62,9 +62,17 @@ class CopyPasteEditProvider implements vscode.DocumentPasteEditProvider {
 
 		// create updated metadata for cell (prep for WorkspaceEdit)
 		const b64string = encodeBase64(fileDataAsUint8);
-		const startingAttachments = currentCell.metadata?.custom?.attachments;
+		const startingAttachments = currentCell.metadata.custom?.attachments;
```

Comment:
Could you add types to prevent future mistakes? Right now `metadata` is `{ [key: string]: any }` but you could add a type and use a cast to make working with it after

Reply:
refactor should keep the typing a bit stricter for the metadata being applied to a cell.